src/op_analytics/transforms/fees/create/02_agg_daily_transactions.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE IF NOT EXISTS _placeholder_
+(
+    `dt` Date,
+    `chain` String,
+    `chain_id` Int32,
+    `network` String,
+    
+    `count_transactions` UInt64,
+    `count_success_transactions` UInt64,
+    `distinct_blocks` UInt64,
+    
+    `sum_l2_gas_used` UInt64,
+    `sum_success_l2_gas_used` UInt64,
+    
+    `sum_tx_fee_native` Int64,
```

Comment:
Same comment:
sum_tx_fee_native and sum_success_tx_fee_native should be Float64 as well, since they're in ETH terms.